fedimint-client/src/lib.rs

```diff
@@ -2214,29 +2159,66 @@ pub async fn get_invite_code_from_db(db: &Database) -> Option<InviteCode> {
 
 /// Tries to download the client config from the federation,
 /// attempts up to `retries` number times
-async fn try_download_config(
-    invite_code: InviteCode,
-    max_retries: usize,
-) -> anyhow::Result<ClientConfig> {
+pub async fn download_client_config(invite_code: &InviteCode) -> anyhow::Result<ClientConfig> {
```

Comment:
Can we make this a constructor/member of `ClientConfig` to replicate the `FederationInfo` interface. I.e. the user should be able to call `ClientConfig::download_with_invite(invite)` or something similar.

Reply:
Done